src/shared/components/palette/address/write-address-field.html

```diff
@@ -8,7 +8,8 @@ <h2 class="heading-h2">{{caseField | ccdFieldLabel }}</h2>
         <span class="form-label">Enter a UK postcode</span>
       </label>
       <span class="error-message" *ngIf="missingPostcode">Enter the Postcode</span>
-      <input type="text" [id]="createId('postcodeInput')" name="postcode" class="form-control postcodeinput" [formControl]="postcode">
+      <input type="text" [ngClass]="{'govuk-input--error': missingPostcode}" [ngStyle]="{'border-width':missingPostcode ? '2px':null}" 
```

Comment:
`[ngStyle]="{'border-width':missingPostcode ? '2px':null}` is 2px border width approved/desired by GDS? If so, could you not do this using classes/ngClass instead of inline styling/ngStyle?

Reply:
As discussed removed the [ngStyle] attribute